WebApp/static/core/js/views/desktop/TaskBar.js

```diff
@@ -5,206 +5,230 @@
-                  } else {
-                    location.href = oHref + '?theme=' + me.text + "&" + GLOBAL.APP.MAIN_VIEW._state_related_url;
-                  }
+              if (oQPosition != -1) {
+                location.href = oHref.substr(0, oQPosition) + '?theme=' + me.text + "&" + GLOBAL.APP.MAIN_VIEW._state_related_url;
+              } else {
+                location.href = oHref + '?theme=' + me.text + "&" + GLOBAL.APP.MAIN_VIEW._state_related_url;
+              }
 
-                }
-              });
-        }
+            }
+          });
+    }
 
-        /*
-         * me.items.push({ xtype : 'tbtext', text : "Theme" });
-         * me.items.push(button_theme);
-         */
+    /*
+     * me.items.push({ xtype : 'tbtext', text : "Theme" });
+     * me.items.push(button_theme);
+     */
 
-        var button_views = {
-          "text" : GLOBAL.VIEW_ID,
-          "menu" : []
-        };
+    var button_views = {
+      "text" : GLOBAL.VIEW_ID,
+      "menu" : []
+    };
 
-        var oListViews = ["desktop", "tabs"];
+    var oListViews = ["desktop", "tabs"];
 
-        for (var i = 0; i < oListViews.length; i++) {
-          button_views.menu.push({
-                text : oListViews[i],
-                handler : function() {
+    for (var i = 0; i < oListViews.length; i++) {
+      button_views.menu.push({
+            text : oListViews[i],
+            handler : function() {
 
-                  var me = this;
+              var me = this;
 
-                  var oHref = location.href;
+              var oHref = location.href;
 
-                  var oQPosition = oHref.indexOf("?");
+              var oQPosition = oHref.indexOf("?");
 
-                  if (oQPosition != -1) {
-                    location.href = oHref.substr(0, oQPosition) + '?view=' + me.text + "&theme=" + sButtonThemeText;
-                  } else {
-                    location.href = oHref + '?view=' + me.text + "&theme=" + sButtonThemeText;
-                  }
+              if (oQPosition != -1) {
+                location.href = oHref.substr(0, oQPosition) + '?view=' + me.text + "&theme=" + sButtonThemeText;
+              } else {
+                location.href = oHref + '?view=' + me.text + "&theme=" + sButtonThemeText;
+              }
 
-                }
-              });
-        }
+            }
+          });
+    }
 
-        me.items.push({
-              xtype : 'tbtext',
-              text : "View"
-            });
+    me.items.push({
+          xtype : 'tbtext',
+          text : "View"
+        });
 
-        me.items.push(button_views);
-        var getAuthCFG = function(Auth = '',Value = ''){
+    me.items.push(button_views);
+    
+    var getAuthCFG = function(Auth){
       var req = Ext.Ajax.request({
         url: GLOBAL.BASE_URL + 'Authentication/getAuthCFG',
-        params: {
+        params: { typeauth: Auth },
+        async: false
+      }).responseText
+      res = JSON.parse(req)
+      console.log(res)
```

Comment:
please remove console.log. If you want to add debug message, please use 
 GLOBAL.APP.CF.log("debug",